runelite-client/src/main/java/net/runelite/client/eventbus/EventBus.java

```diff
@@ -105,6 +107,9 @@ public synchronized void register(@Nonnull final Object object)
 			builder.putAll(subscribers);
 		}
 
+		builder.orderValuesBy(Comparator.comparing(Subscriber::getPriority)
```

Comment:
This should not sort by name, because then it breaks registration order what should be preserved without priority

Reply:
This can be done easily, also abex's idea.